openwisp_monitoring/monitoring/charts.py

```diff
@@ -179,7 +182,7 @@
         'order': 270,
         'query': {
             'influxdb': (
-                "SELECT 100 * MEAN(used_disk) AS disk_usage FROM {key} WHERE "
+                "SELECT 100 * MEAN(used_disk) / MEAN(size_bytes) AS disk_usage FROM {key} WHERE "
```

Comment:
I think this was also better before, I advise avoiding this division.

Reply:
@nemesisdesign, the division was taking previously too. `used_bytes/size_bytes` was getting stored in the `influxdb measurement` earlier (As, we need a percentage for the threshold). Now `size_bytes` and `used_bytes` both are getting stored as individual fields. This might be useful in case we need the usage in `bytes` and not `percentage` in the future but on more thoughts, I think in can be easily changed when the need arises (for now it will just consume space in influxdb :sweat_smile:). Hence, changing it back to how it was before.